rust/src/builders/tx_builder.rs

```diff
@@ -970,6 +970,14 @@ impl TransactionBuilder {
             total_collateral: self.calc_collateral_total()?,
             reference_inputs: self.reference_inputs.as_ref().map(|inputs| TransactionInputs(inputs.iter().map(|utxo| utxo.input.clone()).collect())),
         };
+
+        built.inputs.0.sort_by(|a, b| {
```

Comment:
This isn't enough because the same constraint would apply to all other tag types (mints, withdrawals, etc.)

The choice is to either sort all of these, or make that once you set the redeemers on the tx builder, you can no longer add new inputs/mints/withdrawals/certs (so basically a standalone build step)

Reply:
Done now withdrawals are sorted, mint was already sorted, and certs are not sorted.